src/index.js

```diff
@@ -1,7 +1,11 @@
+require('fullcalendar/dist/fullcalendar.js');
+require('fullcalendar/dist/locale-all.js');
```

Comment:
Why do we need this here? Should be in the module we already import, no?

Reply:
Moved this to the intermediary module